src/inverse-kinematics/include/iDynTree/InverseKinematics.h

```diff
@@ -640,7 +640,7 @@ class iDynTree::InverseKinematics
 
     /*! @name Solution-related methods
       */
-    ///@{
```

Comment:
This removal is breaking the documentation group at https://github.com/robotology/idyntree/pull/353/files#diff-c85a932fafd5e5f39278aee4923d544eL669 .

Reply:
Fixed.